File: StokryptoAndroid/app/src/main/java/app/web/bachbntdev/stokrypto/domain/entities/Asset.kt
package app.web.bachbntdev.stokrypto.domain.entities

interface Asset {
    val id: String
    val symbol: String
    val name: String
    val price: Double
    val volatility: Double
    val volatilityPercent: Double
    val volume: Double
    val type: String
    val unit: String
}

File: StokryptoIOS/Stokrypto/Domain/Entities/Asset.swift
//
//  Asset.swift
//  Stokrypto
//
//  Created by bach.bui on 25/07/2023.
//

import Foundation

protocol Asset {
    var id: String { get }
    var symbol: String { get }
    var name: String { get }
    var price: Double { get }
    var volatility: Double { get }
    var volatilityPercent: Double { get }
    var volume: Double { get }
    var type: String { get }
    var unit: String { get }
}
